README.md

```diff
@@ -21,6 +21,7 @@ Then `bundle install`.
 Here's a quick example, adding the middleware to a Rails app in `config/initializers/omniauth.rb`:
 
 ```ruby
+
```

Comment:
Oh, I didn't realize there was a github site, which explains why it is out of sync since I updated the README in https://github.com/Shopify/omniauth-shopify-oauth2/pull/25 without knowing of its existence